### ISSUE TITLE
delete branches

### ISSUE THREAD

**Issue by acrymble** [programminghistorian/jekyll#769]
I was trying to add the text from #619 but I realised only after creating a new brance (issue-619 & issue-691 because I have fat fingers) that the material was already live. I'm not sure how to delete these two branches, which are no longer needed. @mdlincoln ?

**Comment by ianmilligan1** [comment]
Just click on "branches" on the home page and then the trash can next to the branch you want to delete, @acrymble.

step one
<img width="1180" alt="screen shot 2018-03-30 at 10 22 21 am" src="https://user-images.githubusercontent.com/3834704/38140978-4bccc96a-3404-11e8-8ab3-a9ee29add0ff.png">

step two
<img width="1180" alt="screen shot 2018-03-30 at 10 21 48 am" src="https://user-images.githubusercontent.com/3834704/38140957-30f02e02-3404-11e8-9ff2-02c92249617f.png">

**Comment by acrymble** [comment]
Thanks for the tip!
